src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -127,10 +127,20 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
         const cssTextboxExpiry = style?.cssTextboxExpiry ?? 'width:40px;';
         const csstexboxCvd = style?.csstexboxCvd ?? 'width:40px';
 
+        const monerisLabels: MonerisHostedFieldsLabels = {
+            pan_label: initializationData?.creditCardLabel || 'Credit Card Number',
+            exp_label: initializationData?.expiryDateLabel || 'Expiration',
+            cvd_label: initializationData?.cvdLabel || 'CVD',
+        };
+
+        const labelsQueryString = Object.keys(monerisLabels)
+            .map(key => `${key}=${monerisLabels[key as keyof MonerisHostedFieldsLabels]}`)
+            .join('&');
+
         iframe.name = IFRAME_NAME;
         iframe.id = IFRAME_NAME;
 
-        iframe.src = `${ this._monerisURL(testMode) }?id=${profileId}&pmmsg=true&css_body=${cssBody}&css_textbox=${cssTextbox}&css_textbox_pan=${csstextboxPan}&enable_exp=1&css_textbox_exp=${cssTextboxExpiry}&enable_cvd=1&css_textbox_cvd=${csstexboxCvd}&display_labels=1`;
+        iframe.src = `${ this._monerisURL(testMode) }?id=${profileId}&pmmsg=true&css_body=${cssBody}&css_textbox=${cssTextbox}&css_textbox_pan=${csstextboxPan}&enable_exp=1&css_textbox_exp=${cssTextboxExpiry}&enable_cvd=1&css_textbox_cvd=${csstexboxCvd}&${labelsQueryString}&display_labels=1`;
```

Comment:
The goal with the change above (using `Object.entries` or `map` was to remove this string and make them all just an object. Can we move the rest of the entries in this list to a config object? We can also create a type to make sure the entries correspond with what's expected in the url.

```ts
interface MoneriesQueryParams {
}

const queryParams = {
	id: profileId,
    pmmsg: true,
    // ...
    display_labels: 1,
};

const queryString = map(queryParams, (value, key) => `${key}=${value}`).join('&');

iframe.src = `${ this._monerisURL(testMode) }?${ queryString }`;
```